src/Action/EntrypointAction.php

```diff
@@ -9,12 +9,12 @@
  * file that was distributed with this source code.
  */
 
-namespace ApiPlatform\Core\Hydra\Action;
+namespace ApiPlatform\Core\Action;
```

Comment:
> Schema.org itself contains this concept: https://schema.org/EntryPoint

No. That's a different thing, if you look carefully. Their naming is just confusing sometimes... :laughing: